cli/pack/systemd_dir.go

```diff
@@ -0,0 +1,112 @@
+package pack
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/util"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	defaultInstanceFdLimit = 65535
+)
+
+//go:embed templates/app-unit-template.txt
```

Comment:
Please, use file extension to enable code highlight in text editor: .conf or .ini.

Reply:
Fixed.